Formula/grepip.rb

```diff
@@ -0,0 +1,28 @@
+class Grepip < Formula
+  desc "Filters IPv4 & IPv6 addresses with a grep-compatible interface"
+  homepage "https://ipinfo.io"
+  url "https://github.com/ipinfo/cli/archive/grepip-1.0.2.tar.gz"
+  sha256 "f9258f9f5c2bfdb4278c6cd456072664be7588fc6a8b665d35cbd6fa29f57568"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "./grepip/build.sh"
+    bin.install "build/grepip"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "./grepip"
```
I think this is basically the same thing that your build script runs. Let's do this instead, as it's a bit more transparent about what the build is doing exactly here.

Reply:
Hey @carlocab,

The main issue I have with this is it means changes to the build steps in the repo require changes here. Granted this wouldn't happen often, but I would still really prefer keeping the build details wrapped if possible.

What are your thoughts?

Reply:
>changes to the build steps in the repo require changes here

This is actually another reason why I want to make these steps explicit here. We'd like the opportunity to review changes to the build instead of applying them blindly.

Reply:
I understand, but I'm just a little confused about it because there seem to be thousands of formulae running e.g. `make`, which effectively hides the details as well. I'm happy to make the change, but I'm confused why this is particularly important for this formula.

Reply:
That's true, but it's a lot harder to put the contents of a `Makefile` in the `install` method. We can do this here, so we may as well.

Reply:
Alright let's do it then.